### PR TITLE
Update for new template system

### PR DESCRIPTION
**Pull request by tmorrisonsfx**
What changed?
- Add info about vSphere requirements [DOCS-1580](https://signalfuse.atlassian.net/browse/DOCS-1580)
- Add info about vSphere monitoring setup [DOCS-1324](https://signalfuse.atlassian.net/browse/DOCS-1324)
- To do this, I had to use the new template system to update the Installation section.
- Add README.md.jinja - done
- Update meta.yaml - done
- Delete original (non-Jinja) README.md - done

### CODE REVIEW

**Comment by tmorrisonsfx** [comment]
@pmcollins Hi Pablo, can you review this PR? I added a note about the permissions in the Installation section of the README.md.jinja file.

File: vsphere/README.md.jinja [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+After this initial sweep, the monitor will query the vCenter for performance data and metrics. This query takes place every 20 seconds, 
+which is the interval at which the vCenter makes real-time performance data available. As a result, regardless of the `intervalSeconds` 
+value in the agent config, this monitor will run every 20 seconds.
+
+The monitor also refreshes, at a configurable interval, the cache of hosts, virtual machines, and metrics. By default, this refresh takes 
+place every 60 seconds; however, this interval can be changed by updating the configuration field `InventoryRefreshInterval`.
+
+#### Compatibility
+
+This monitor uses VMware’s govmomi SDK, which officially supports vCenter 6.0, 6.5, 6.7, and 7.0. While this monitor may work with
```
Comment (pmcollins) [inline vsphere/README.md.jinja:38]: Can we go with official support for 6.5, 6.7, and 7.0 and unofficial support for 5.1, 5.5 and 6.0?

I'm using this as a reference: https://github.com/vmware/govmomi#compatibility

**Review by pmcollins — APPROVED** [review]
Just one comment below, otherwise LGTM.

**Comment by tmorrisonsfx** [comment]
@keitwb Can I merge this with the failed CircleCI checklinks test? It's failing due to this missing file: Stat ../_sidebars-and-includes/smart-agent-access-token.html. If that file has been removed, does it still need to be part of the checklinks test? 

Cc @jchengsfx

**Pull request merged**